racingcar-java/src/main/java/racingcar/Car/Cars.java

```diff
@@ -0,0 +1,29 @@
+package racingcar.Car;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+public class Cars {
+    private static final Random random = new Random();
+    private static final int MAX_BOUND = 10;
+
+    private List<Car> cars;
```

Comment:
```suggestion
    private final List<Car> cars;
```
cars 클래스의 참조값을 final로 두면 혹시나 Cars객체를 손댈 때 참조값이 변경됨을 방지할 수 있어요